backend/webserver/serializers.py

```diff
@@ -1,12 +1,17 @@
 from rest_framework import serializers
-from .models import Author
+from .models import Author, Post
 
 
 class AuthorSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Author
         fields = ['url', 'id', 'display_name', 'profile_image', 'github_handle']
 
+class PostSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Post
+        fields = ['author','created_at','edited_at','title','description','source','origin','unlisted']
```

Comment:
Where is `content_type` and `content`? Are we following the output format as specified here - https://github.com/zarifmahfuz/project-socialdistribution/blob/master/project.org#post ?